### PR TITLE
Use FINE not FINEST for log messages that used to be INFO

### PR DESCRIPTION
**Pull request by jglick**
Minor amendment to https://github.com/jenkinsci/jenkins/pull/4345 to make it more friendly to e.g. https://github.com/jenkinsci/parameterized-trigger-plugin/pull/169.

### Proposed changelog entries

* N/A

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- [x] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [x] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Review by MarkEWaite — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]
🤷

**Review by res0nance — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
